sdcm/utils/common.py

```diff
@@ -1184,6 +1184,64 @@ def can_connect_to(ip: str, port: int, timeout: int = 1) -> bool:
     return result == 0
 
 
+def find_scylla_repo(scylla_version, dist_type='centos', dist_version=None):
+    """
+    Get a repo/list of scylla, based on scylla version match
+
+    :param scylla_version: branch version to look for, ex. 'branch-2019.1:latest', 'branch-3.1:l'
+    :param dist_type: one of ['centos', 'ubuntu', 'debian']
+    :param dist_version: family name of the distro version
+    :raises: ValueError if not found
+
+    :return: str url repo/list
+    """
+    if ':' in scylla_version:
+        branch_repo = get_branched_repo(scylla_version)
```

Comment:
I think you should pass `dist_type` here...
add unit-test to check dist_type

Reply:
No need, since we now have unified deb package, it's the same package for ubuntu/debian, regardless of their versions, like it used to be.

Reply:
scratch that, you are correct...

Reply:
fixed